src/main/scala/shade/memcached/internals/Status.scala

```diff
@@ -1,5 +1,7 @@
 package shade.memcached.internals
 
+import net.spy.memcached.ops.OperationStatus
+
 sealed trait Status
```

Comment:
Since we are on this `Status`, make this `extends Product with Serializable`, it doesn't hurt and it helps the compiler with type inference :-)